package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Generates random dessert names from a precompiled list in code. No database is used.",
   "main": "index.js",
   "scripts": {
-    "test": "node ./node_modules/mocha/bin/mocha"
+    "test": "mocha --recursive"
```

Comment:
The `test` command is fix like this, to run the tests in the `test` directory recursively.